### PR TITLE
Add possibility to provide path as argument and default value 

### PR DESCRIPTION
**Pull request by racicLuka**
Added possibility to call the script with an argument:
`./robotmk_collector.ps1 "path_as_argument"`

Also, if `env:MK_CONFDIR` is not present, the path will default to `C:\ProgramData\checkmk\agent\config\robotmk.json`

### CODE REVIEW

File: checkmk_extensions/agent/robotmk_collector.ps1
Comment (jherbel) [inline checkmk_extensions/agent/robotmk_collector.ps1:10]: Let's move the new code into a separate function

File: checkmk_extensions/agent/robotmk_collector.ps1
Comment (jherbel) [inline checkmk_extensions/agent/robotmk_collector.ps1:6]: Since we are never calling `Main` with a parameter, we can remove it alltogether

File: checkmk_extensions/agent/robotmk_collector.ps1
Comment (jherbel) [inline checkmk_extensions/agent/robotmk_collector.ps1:18]:
```
$configDir = $env:MK_CONFDIR
if (!$configDir) {
    $configDir = 'C:\ProgramData\checkmk\agent\config'
}
return $configDir + '\robotmk.json'
```

**Review by jherbel — CHANGES_REQUESTED** [review]
Let's also remove the comments, they shouldn't be necessary

File: checkmk_extensions/agent/robotmk_collector.ps1
Comment (jherbel) [inline checkmk_extensions/agent/robotmk_collector.ps1:29]: I think this should be `String []`

File: checkmk_extensions/agent/robotmk_collector.ps1
Comment (jherbel) [inline checkmk_extensions/agent/robotmk_collector.ps1:28]:
```
Param (
        [parameter(Mandatory=$true, Position=0)]
```

File: checkmk_extensions/agent/robotmk_collector.ps1
Comment (jherbel) [inline checkmk_extensions/agent/robotmk_collector.ps1:29]: Let's call it `$CommandLineArgs` or so, since we don't know yet if there actually is a path

File: checkmk_extensions/agent/robotmk_collector.ps1
Comment (jherbel) [inline checkmk_extensions/agent/robotmk_collector.ps1:75]: This is not needed

**Review by jherbel — CHANGES_REQUESTED** [review]
Please remove the code comments and squash the PR into a single commit

**Comment by racicLuka** [comment]
I can squash and merge once you approve the PR, if that is OK with you?

**Comment by jherbel** [comment]
> I can squash and merge once you approve the PR, if that is OK with you?

Depends on how the commit would look afterwards. The easiest way would be to simply squash locally and do a force push.

File: checkmk_extensions/agent/robotmk_collector.ps1
Comment (jherbel) [reply checkmk_extensions/agent/robotmk_collector.ps1]: What I meant:
`[String []]$CommandLineArgs`
So an array of strings

**Review by jherbel — COMMENTED** [review]
See the last remaining comment about `[String []]$CommandLineArgs`

File: checkmk_extensions/agent/robotmk_collector.ps1
Comment (racicLuka) [inline checkmk_extensions/agent/robotmk_collector.ps1:34]: I needed to change this to `$CommandLineArgs -ne ""` because now, if there are no arguments provided, it considers it as an empty string.

**Review by jherbel — APPROVED** [review]

**Pull request merged**
